recipes-devtools/deepstream/deepstream-6.0_6.0.0-1.bb

```diff
@@ -104,7 +97,6 @@ FILES:${PN}-samples-data = "\
 FILES:${PN}-sources = "${DEEPSTREAM_PATH}/sources"
 
 FILES:${PN}-iothub-client = "${DEEPSTREAM_PATH}/lib/libiothub_client.so"
-FILES:${PN}-azure-edge = "${DEEPSTREAM_PATH}/lib/libazure*"
```

Comment:
I think we still need the `-azure-edge` package, but the libraries to put there are `libnvds_azure*`.

Reply:
Yes, you're right.
I will update the recipe